channels/cl.m3u

```diff
@@ -59,8 +57,6 @@ https://unlimited1-cl-movistar.dps.live/lared/lared.smil/playlist.m3u8
 https://server1.oklanet.cl:1936/maximavideo1/maximavideo1/playlist.m3u8
 #EXTINF:-1 tvg-id="Mega.cl" tvg-country="CL" tvg-language="" tvg-logo="" group-title="",Mega
 http://186.67.117.178:8081/play/a00x
-#EXTINF:-1 tvg-id="Mega.cl" tvg-country="CL" tvg-language="" tvg-logo="" group-title="",Mega (720p) [Offline]
-https://mdstrm.com/live-stream-playlist/561430ae330428c223687e1e.m3u8
```

Comment:
https://mdstrm.com/live-stream-playlist/53d2c1a32640614e62a0e000.m3u8 Geolocked

Reply:
> https://mdstrm.com/live-stream-playlist/53d2c1a32640614e62a0e000.m3u8 Geolocked

via VPN also gives an error
![image](https://user-images.githubusercontent.com/7253922/133593013-e17d881b-ef36-408a-a4e2-e872d676a2d4.png)

Reply:
Content with restricted access... might be because of copyright issues.